lib/hcloud/volume.rb

```diff
@@ -18,6 +18,8 @@ class Volume
     has_actions
 
     def attach(server:, automount:)
```

Comment:
Although not documented the default for `automount` is `false`, so I would reflect that default here as well.